server.js

```diff
@@ -15,9 +15,9 @@ app.get('/', (req, res) => {
   res.send('smoke test');
 });
```

Comment:
Once you revert the naming of the routes, this will become an issue again. Can you figure out how to solve this issue?